### ISSUE TITLE
A question for your codes?

### ISSUE THREAD

**Issue by Keysmis** [nxsEdson/MLCR#1]
I'm wondering what the three values mean? 
Is_begin_weight,begin_weight1,begin_weight2
thx~

**Comment by nxsEdson** [comment]
We need the pre-trained weights as the initialization of  GCN. These values are used for controlling the initialization.

**Comment by Keysmis** [comment]
and what process method for the label is 999?

**Comment by Keysmis** [comment]
and the variable self.weight is how to get it?  is it get from the annotated labels by computing the  overall positive/negative of each AU?

**Comment by nxsEdson** [comment]
For unlabeled data, I just ignore them and do not back pass the loss. For self.weight, do you mean the adjacent map for GCN? I calculate the map based on the whole database.

**Comment by Keysmis** [comment]
@nxsEdson Thanks for your great work!, yeah, i already compute that weight in whole dataset, but i have another question......， i couldn't found the variable of self.select_sample in computing the JS dis... ， what does that mean?
and the weights of BCE_sigmoid_negtive_bias_all loss function, is it compute the pos/neg ratio in whole dataset?
